### PR TITLE
Addressing some minor (easy) pre-telechat comments 

### PR DESCRIPTION
**Pull request by furry13**
From:
-Peter Yee
- Mallory Knodel 
- James Guichard
- Gunter Van de Velde

### CODE REVIEW

**Review by wkumari — APPROVED** [review]
THANK YOU!!!!!!!

**Review by wkumari — APPROVED** [review]
Also thank you!!!!!

**Comment by wkumari** [comment]
Seriously, thank you!

File: draft-ietf-dhc-addr-notification.md [hunk 1]
```diff
@@ -391,6 +391,6 @@ This document introduces the following new entities which require an allocation
 # Acknowledgments
 {:numbered="false"}
 
-Many thanks to Bernie Volz for significant review and feedback, as well as Hermin Anggawijaya, Brian Carpenter, Stuart Cheshire, Alan DeKok, Ryan Globus, Erik Kline, David Lamparter, Ted Lemon, Eric Levy-Abegnoli, Aditi Patange, Jim Reid, Michael Richardson, Mark Smith, Eric Vyncke, Timothy Winters for their feedback, comments and guidance. We apologize if we inadvertently forgot to acknowledge anyone's contributions.
+Many thanks to Bernie Volz for significant review and feedback, as well as Hermin Anggawijaya, Brian Carpenter, Stuart Cheshire, Alan DeKok, James Guichard, Ryan Globus, Erik Kline, Mallory Knodel, David Lamparter, Ted Lemon, Eric Levy-Abegnoli, Aditi Patange, Jim Reid, Michael Richardson, Mark Smith, Gunter Van de Velde, Eric Vyncke, Timothy Winters, Peter Yee for their feedback, comments and guidance. We apologize if we inadvertently forgot to acknowledge anyone's contributions.
```
Comment (lcolitti) [inline draft-ietf-dhc-addr-notification.md:394]: Nit: Guichard sorts after Globus.

**Pull request merged**